sympy/simplify/radsimp.py

```diff
@@ -555,6 +555,27 @@ def collect_const(expr, *vars, **kwargs):
     >>> collect_const(2*x - 2*y - 2*z, -2)
     2*x - 2*(y + z)
 
+    ---->> According to documentation, we have the second argument as *vars.
+          *vars - implies that you can pass on as many arguments as possible.
+
+    Examples -- 
```

Comment:
There's no need to repeat "examples" here.

Also, you should use RST formatting. See https://github.com/sympy/sympy/wiki/Writing-documentation.